src/components/IconV2/IconV2-story.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  iconAdd,
+  iconAddGlyph,
```

Comment:
Would this be `iconAdd` and `glyphAdd` versus `iconGlyphAdd`? From a naming convention perspective, that is. Totally get that's what is exported from that package currently 😄

Reply:
@asudoh - going to merge this PR once this comment is addressed 👍

Reply:
Hi @joshblack trying to understand what you meant here - Are you suggesting the names of exports in `carbon-icons` (that’s auto-generated from icon IDs with `s.replace(/\-+([A-z])/g, (match, token) => token.toUpperCase())` logic)? Or something else?

Reply:
Yeah, I think this requires a change in the exports of `carbon-icons`, in this case. I'm not fully aware of all the icon permutations though so might be better to have a naming issue to discuss this kind of stuff just so everyone can weigh in.

Reply:
Created: https://github.com/carbon-design-system/carbon-icons/issues/59